src/pyskindose/table_data/plot_HVL_data.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
+from pathlib import Path
+import os
+import plotly.graph_objects as go
+
+
+data_path = Path(os.path.abspath(__file__)).parent
```

Comment:
Shouldn't this whole file be changed into a function so that it is not run as soon as the file is imported

Reply:
plot script removed, we dont need it